python/setup.py

```diff
@@ -96,7 +96,7 @@ def finalize_options(self):
     packages=["tskit"],
     include_package_data=True,
     ext_modules=[_tskit_module],
-    install_requires=[numpy_ver, "h5py", "svgwrite", "jsonschema"],
+    install_requires=[numpy_ver, "h5py", "svgwrite", "jsonschema", "attrs"],
```

Comment:
The metadata branch uses `kw_args` so I've pinned to that version. I've checked attrs' changelog and releases since then have nothing relevant. https://github.com/tskit-dev/tskit/pull/553/commits/986248416d5129da50ddd7b78285df1fd8ef2ade